tests/e2e/specs/drawer.cy.js

```diff
@@ -22,6 +22,9 @@ describe('Drawer component', () => {
       .get('.v-navigation-drawer')
       .should('be.visible')
   })
+  it('it should have a width of 360', () => {
+    cy.get('.v-navigation-drawer').invoke('innerWidth').should('be.lt', 360)
```

Comment:
Why does the test say "have a width of 360" but then checks that it is less than that?

Reply:
Will fix this with issue below